oaknational/Oak-Web-Application
# Fix/hubspot errors

## Description

- creates a hook to submit to hubspot separate from performing downloads

## Fixes
There seems to be a bug with hubspot submissions failing and causing the downloads page to show an error message even though the download succeeded. The error with hubspot is still occurring but now it is not blocking the downloads journey, and should be clearer in bugsnag.

## How to test
I can reproduce this error consistently on firefox (mac) but not on chrome

To reproduce the error:
1. go to https://thenational.academy/teachers/programmes/citizenship-secondary-ks3-l/units/citizenship-whats-it-all-about-e038/lessons/what-is-citizenship-c4t38d/downloads?preselected=all
2. click Download
3. see the error 

To test the fix:
1. Go to https://deploy-preview-2155--oak-web-application.netlify.thenational.academy/teachers/programmes/citizenship-secondary-ks3-l/units/citizenship-whats-it-all-about-e038/lessons/what-is-citizenship-c4t38d/downloads?preselected=all
4. Click on Download
5. You should see the download succeed and page update to confirmation state
